src/playlist-loader.js

```diff
@@ -227,6 +227,20 @@ const PlaylistLoader = function(srcUrl, hls, withCredentials) {
     }
   };
 
+    // returns the number of enabled playlists on the master playlist object
+  loader.enabledPlaylists = function() {
+    let playlists = loader.master.playlists;
+    let count = 0;
+    let i;
+
+    for (i = 0; i < playlists.length; i++) {
```

Comment:
This could be more concisely written with a `filter` and returning the resultant array's length.

You should avoid using plain-ol'-`for` loops whenever possible.